Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -0,0 +1,5783 @@
+    """Delete a protected hostname.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    name = args['name']
+    response = client.protected_hostname_delete_request(name)
+    command_results = generate_simple_command_results('name', name, response,
+                                                      OutputTitles.PROTECTED_HOSTNAME_GROUP_DELETE)
+    return command_results
+
+
+def protected_hostname_group_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Get protected hostname list / object.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    protected_hostname = args.get('name')
+    response = client.protected_hostname_list_request(name=protected_hostname)
+    parsed_data, pagination_message, formatted_response = list_response_handler(
+        client=client,
+        response=response,
+        data_parser=client.parser.protected_hostname_group,
+        args=args,
+        sub_object_id=protected_hostname,
+        sub_object_key='_id' if client == ClientV1.API_VER else 'name')
+    headers = create_headers(client.version, ['id', 'default_action', 'protected_hostname_count'], ['can_delete'], [])
+    readable_output = tableToMarkdown(name=OutputTitles.PROTECTED_HOSTNAME_GROUP_LIST,
+                                      metadata=pagination_message,
+                                      t=parsed_data,
+                                      headers=headers,
+                                      headerTransform=string_to_table_header)
+    command_results = CommandResults(readable_output=readable_output,
+                                     outputs_prefix='FortiwebVM.ProtectedHostnameGroup',
+                                     outputs_key_field='id',
+                                     outputs=parsed_data,
+                                     raw_response=response)
+    return command_results
+
+
+def protected_hostname_member_validation(args: Dict[str, Any]):
+    """Protected hostname member args validator.
+
+    Args:
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Raises:
+        DemistoException: Errors.
+    """
+    if args.get('action') and args['action'] not in ['Allow', 'Deny', 'Deny (no log)']:
+        raise DemistoException(ERRORS.ACTION)
+    if args.get('ignore_port') and args['ignore_port'] not in ['enable', 'disable']:
+        raise DemistoException(ERRORS.IGNORE_PORT)
+    if args.get('include_subdomains') and args['include_subdomains'] not in ['enable', 'disable']:
+        raise DemistoException(ERRORS.INCLUDE_SUBDOMAINS)
```

Comment:
Why did you chose to raise **DemistoException**? Isn't ValueError Exception will be more suitable for these cases?